hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioNetworking.java

```diff
@@ -147,6 +143,10 @@ public void start() {
 
         logger.log(selectorMode != SELECT ? INFO : FINE, "IO threads selector mode is " + selectorMode);
 
+        if (metricsRegistry.minimumLevel().isEnabled(DEBUG)) {
```

Comment:
You are now undoing this PR:

https://github.com/hazelcast/hazelcast/pull/14698/files